extension/js/agent/utils/knownTypes.js

```diff
@@ -183,8 +183,8 @@
       });
     }
 
-
-    if (Agent.patchedBackbone.Wreqr) {
+    //Marionette 2.0.0 does not have Radio in Wreqr
+    if (Agent.patchedBackbone.Wreqr && Agent.patchedBackbone.Wreqr.Channel) {
```

Comment:
Now simply checks if Wreqr.Channel exits before trying to add it as a known type